synapse/lib/view.py

```diff
@@ -763,7 +1011,7 @@ async def setViewInfo(self, name, valu):
         '''
         Set a mutable view property.
         '''
-        if name not in ('name', 'desc', 'parent', 'nomerge'):
+        if name not in ('name', 'desc', 'parent', 'nomerge', 'quorum'):
```

Comment:
When setting a `parent` value should we also verify that the parent has no open merge requests?

Reply:
good catch, thanks!